### PR TITLE
Added back to dev command for releases

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Added command to set versions back to dev on a repo

# Need Review By (Date)
- soon

# Urgency
- medium

# Steps to Test
1. try out the changes that are part of the `config/.circleci/config.yml` file

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: config/.circleci/config.yml [hunk 1]
```diff
@@ -57,16 +57,38 @@ behat: &behat
     - store_artifacts:
         path: /var/www/html/artifacts
 
+back_to_dev: &back_to_dev
+  <<: *defaults
+  steps:
+    - checkout
+    - run:
+        name: Run Behat Tests
```
Comment (sherakama) [inline config/.circleci/config.yml]:
```suggestion
        name: Back to dev
```

File: src/Robo/Tasks/RoboFile.php [hunk 2]
```diff
@@ -22,6 +22,45 @@ class RoboFile extends Tasks {
...
+   * @param string $directory
+   *   Directory where the module is to be updated.
+   *
+   * @command back-to-dev
+   * @usage vendor/bin/sws-caravan back-to-dev 8.1.0 /var/www/stanford_module
+   * @usage vendor/bin/sws-caravan back-to-dev ${CIRCLE_TAG} ${CIRCLE_WORKING_DIRECTORY}
+   */
+  public function backToDev($old_semver, $directory) {
+    list($major, $minor, $point) = explode('.', $old_semver);
+    $info_yamls = $this->rglob("$directory/*.info.yml");
```
Comment (sherakama) [inline src/Robo/Tasks/RoboFile.php:39]: This is probably safe. I cannot think of anything else that uses this convention.

**Comment by sherakama** [comment]
I haven't actually ran it but it looks like it should work.

**Review by imonroe — APPROVED** [review]
These look good to me.

**Review by sherakama — APPROVED** [review]

**Pull request merged**